hadoop-ozone/ozone-manager/src/test/java/org/apache/hadoop/ozone/om/request/s3/bucket/package-info.java

```diff
@@ -0,0 +1,23 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ *  with the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+/**
+ * Package contains test classes for s3 bucket requests.
+ */
+package org.apache.hadoop.ozone.om.request.s3.bucket;
```

Comment:
Add newline at end of file.

Reply:
Done

Reply:
Sorry I still don't see a newline. It's a minor point. See https://unix.stackexchange.com/questions/18743/whats-the-point-in-adding-a-new-line-to-the-end-of-a-file

This is an old habit from using a C++ compiler that would throw errors on files not ending in newlines. 🙂

Reply:
Added the new line, in my IDE is see it, not sure GitHub UI is not showing that.

![Screen Shot 2019-07-17 at 4 42 01 PM](https://user-images.githubusercontent.com/8586345/61419169-dce23200-a8b1-11e9-8661-ee7813ded2e3.png)

Reply:
![Screen Shot 2019-07-17 at 4 42 01 PM](https://user-images.githubusercontent.com/8586345/61419182-f5eae300-a8b1-11e9-8863-5a7f8c874f03.png)

Reply:
![Screen Shot 2019-07-17 at 4 43 02 PM](https://user-images.githubusercontent.com/8586345/61419202-03a06880-a8b2-11e9-9ba9-11428b9d96bf.png)